src/components/ha-card.ts

```diff
@@ -0,0 +1,48 @@
+import "@polymer/paper-styles/element-styles/paper-material-styles";
+import {
+  html,
+  TemplateResult,
+  PropertyDeclarations,
+  LitElement,
+} from "lit-element";
+
+class HaCard extends LitElement {
+  protected render(): TemplateResult | void {
+    return html`
+      <style include="paper-material-styles">
+        :host {
```

Comment:
Do the CSS like this: https://github.com/home-assistant/home-assistant-polymer/blob/7a16961387e1be5e0b2826c722d732b09548dbd0/src/panels/lovelace/components/hui-generic-entity-row.ts#L88-L90